toys-core/lib/toys/compat.rb

```diff
@@ -23,7 +28,7 @@ def self.windows?
 
     # @private
     def self.allow_fork?
-      !jruby? && !windows?
+      !jruby? && !truffleruby? && !windows?
```

Comment:
A better way to check is `Process.respond_to?(:fork)`
https://github.com/oracle/truffleruby/blob/master/doc/user/compatibility.md#fork